src/components/Articles.jsx

```diff
@@ -1,4 +1,9 @@
 import React, { useState, useEffect } from 'react';
+import Moment from 'react-moment';
+// import 'moment-timezone';
+
+// TODO - Links for username, title and buttons for removing and viewing
+
```

Comment:
You have a catch block in you api function so we will catch the error before we get to the component and never catch it in the component.